earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ ! -d Imagery ]; then
+	wget http://data.opengee.org/FusionTutorial-Full.tar.gz -O /tmp/FusionTutorial-Full.tar.gz
```

Comment:
Minor: it might be a good idea to make `/tmp/FusionTutorial-Full.tar.gz` a variable.